lib/foreman_tasks/tasks/cleanup.rake

```diff
@@ -3,21 +3,21 @@ namespace :foreman_tasks do
     desc <<DESC
 Clean tasks based on filter and age. ENV variables:
 
-  * FILTER     : scoped search filter (example: 'label = "Actions::Foreman::Host::ImportFacts"')
-  * AFTER      : delete tasks created after *AFTER* period. Expected format is a number followed by the time unit (s,h,m,y), such as '10d' for 10 days
-  * STATES     : comma separated list of task states to touch with the cleanup, by default only stopped tasks are covered
-  * NOOP       : set to "true" if the task should not actuall perform the deletion
-  * VERBOSE    : set to "true" for more verbose output
-  * BATCH_SIZE : the size of batches the tasks get processed in (1000 by default)
+  * TASK_SEARCH : scoped search filter (example: 'label = "Actions::Foreman::Host::ImportFacts"')
+  * AFTER       : delete tasks created after *AFTER* period. Expected format is a number followed by the time unit (s,h,m,y), such as '10d' for 10 days
+  * STATES      : comma separated list of task states to touch with the cleanup, by default only stopped tasks are covered
+  * NOOP        : set to "true" if the task should not actuall perform the deletion
+  * VERBOSE     : set to "true" for more verbose output
+  * BATCH_SIZE  : the size of batches the tasks get processed in (1000 by default)
 
-If none of FILTER, BEFORE, STATES is specified, the tasks will be cleaned based
+If none of TASK_SEARCH, BEFORE, STATES is specified, the tasks will be cleaned based
 configuration in settings
 DESC
     task :run => 'environment' do
       options = {}
 
-      if ENV['FILTER']
```

Comment:
You mean like a deprecation error (e.g. `Please use TASK_SEARCH instead of FILTER`)?